### ISSUE TITLE
office-addin-dev-certs verify fails on windows

### ISSUE THREAD

**Issue by tobiasdiez** [OfficeDev/Office-Addin-Scripts#769]
# Prerequisites

Please answer the following questions before submitting an issue. 
**YOU MAY DELETE THE PREREQUISITES SECTION.**
- [x] I am running the latest version of Node and the tools: Node 19, office-addin-dev-certs 1.9.2
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

# Expected behavior

Please describe the behavior you were expecting

Running `verify` directly after `install` reports success.

# Current behavior

It fails with "You need to install certificates for trusted access to https://localhost.".

## Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Run `yarn office-addin-dev-certs install --machine` with admin rights
2. Run `yarn office-addin-dev-certs verify`

## Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Operating System: Windows 11
* Node version: 19
* Tool version: 1.9.2

## Failure Logs

It's also not only that the verification is failing, but word and firefox both complain that the certificate for localhost cannot be verified. Only solution I've currently found is to manually install the `ca.crt` file in "LocalMachine/Trusted Root Certification Authorities" (it no longer works if I also install the localhost.crt file). Quite confusing.

```bash
> Get-ChildItem cert:\\CurrentUser\\Root | Where-Object Issuer -like "*CN=Developer*" | Where-Object { $_.NotAfter -gt [datetime]::today.AddDays(-1) } | Format-List

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : A93C6CE2CC75383C07818B1305FF8A34AE93A26E
FriendlyName :
NotBefore    : 21/04/2023 20:13:58
NotAfter     : 21/05/2023 20:13:58
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : 73D793CFD0690F06EFA095A3824D1B43FDA36168
FriendlyName :
NotBefore    : 21/04/2023 20:52:27
NotAfter     : 21/05/2023 20:52:27
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : 6F3D8B2295412D5B6B0EFF83ECF2719E93CCF0A6
FriendlyName :
NotBefore    : 21/04/2023 20:27:43
NotAfter     : 21/05/2023 20:27:43
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : 41BB28846D3F7BDCD83286AC5DDCBEDA995453B8
FriendlyName :
NotBefore    : 21/04/2023 20:47:20
NotAfter     : 21/05/2023 20:47:20
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}

>  Get-ChildItem cert:\\LocalMachine\\Root | Where-Object Issuer -like "*CN=Developer*" | Where-Object { $_.NotAfter -gt [datetime]::today.AddDays(-1) } | Format-List

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : A93C6CE2CC75383C07818B1305FF8A34AE93A26E
FriendlyName :
NotBefore    : 21/04/2023 20:13:58
NotAfter     : 21/05/2023 20:13:58
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : 73D793CFD0690F06EFA095A3824D1B43FDA36168
FriendlyName :
NotBefore    : 21/04/2023 20:52:27
NotAfter     : 21/05/2023 20:52:27
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : 6F3D8B2295412D5B6B0EFF83ECF2719E93CCF0A6
FriendlyName :
NotBefore    : 21/04/2023 20:27:43
NotAfter     : 21/05/2023 20:27:43
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}

Subject      : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Issuer       : O=Developer CA for Microsoft Office Add-ins, L=Redmond, S=WA, C=US, CN=Developer CA for Microsoft Office Add-ins
Thumbprint   : 41BB28846D3F7BDCD83286AC5DDCBEDA995453B8
FriendlyName :
NotBefore    : 21/04/2023 20:47:20
NotAfter     : 21/05/2023 20:47:20
Extensions   : {System.Security.Cryptography.Oid, System.Security.Cryptography.Oid}
```

**Comment by millerds** [comment]
Does the same thing happen when you don't use the '--machine' option to install the certificate?  Can you try using a more up to date version of office-addin-dev-certs (1.11.3 is the current version)?

BTW: Office developer tools support running on the LTS branches of node . . . which is v18.  This shouldn't have anything to do with this in particular (since it's powershell commands), but technically it is an unsupported setup.

**Comment by tobiasdiez** [comment]
Oh, I thought I've already updated office-addin-dev-certs. It is fixed with version 1.11.3. 🚀 Sorry for the noise.
